persistence-modules/hibernate-annotations/src/test/java/com/baeldung/hibernate/creationupdatetimestamp/HibernateCreationUpdateTimestampIntegrationTest.java

```diff
@@ -71,22 +72,22 @@ void whenCreatingEntity_ThenCreatedOnAndLastUpdatedOnAreBothSet() {
     }
 
     @Test
-    void whenCreatingEntity_ThenCreatedOnAndLastUpdatedOnAreNotEqual() {
+    void whenCreatingEntity_ThenCreatedOnAndLastUpdatedOnAreEqual() {
```

Comment:
I would say that the test itself was wrong since the timestamp cannot be different since was initialized in just a single place and were initialized once.

Reply:
yeah that could work, actually there could be a milliseconds difference b/w creation and update timestamp, let's assume they would be equal for now